RFC/src/RFC-0110_BaseNodes.md

```diff
@@ -138,6 +138,44 @@ Valid transactions are:
 * added to the [mempool];
 * forwarded to peers using the transaction [BroadcastStrategy].
 
+#### Block/Transaction Weight 
+[block-transaction weight]: #blocktransaction-weight "Block/Transaction Weight"
+
+The weight of a transaction / block measured in "grams". Input, output and kernel weights reflect their respective relative 
+storage and computation cost. Transaction fees are typically proportional to a transaction body's total weight, creating 
+incentive to reduce the size of the UTXO set.
+
+|                          	| Bytes    	| Weight (grams) 	| Notes                                                               	|
+|--------------------------	|----------	|----------------	|---------------------------------------------------------------------	|
+| Output:                  	|          	|                	|                                                                     	|
+| - Per output             	| ~777     	| 13             	| Incl. Rangeproof                                                    	|
+| - Asset Registration     	| >= 256   	| 4              	| Set in OutputFeatures, size depends on number. of template IDs used 	|
+| Input:                   	|          	|                	|                                                                     	|
+| - Per input              	| ~196     	| 1              	| Excl. script field                                                  	|
+| - Tari script per opcode 	| variable 	| 1              	| Weighting for computation at at spend time                          	|
```

Comment:
There was a discussion about saving the entire script and not just the hash of the script on the output. Should we not move the script cost to the output then?

We don't want to "penalize" you spending a script, aka pruning it, but we want to penalize you for saving a large. script.

Reply:
Yes, that sounds 100% correct - will change